docs/src/modules/ROOT/pages/enterprise-edition/enterprise-edition.adoc

```diff
@@ -86,6 +87,40 @@ repositories {
 }
 ----
 --
+
+Pyproject.toml::
```

Comment:
I question if we want to do this.

We don't want to push Python to Enterprise users, at least not yet.
Having documentation for it suggests that Timefold will support it; this is not the case.

Reply:
We could add a note saying "Support is currently not included with Timefold Solver for Python Enterprise Edition"?

Reply:
"Timefold does not offer Enterprise support for Python yet."